CONTRIBUTING.md

```diff
@@ -0,0 +1,28 @@
+# How to contribute
+
+First of all, **thanks for contributing**! Here are some guidelines to make it happen in the best way possible.
+
+## Opening an issue
+
+When you open an issue, please be as precise as possible :
+- if it's a bug, how can we reproduce it ? On which environment are your running ?
+- if it's a feature request, could you give us some examples of what you'd like ?
+
+Please use labels to explicit the kind of ticket it is. Do not fill up any assignee or milestone, we'll set it after 
```

Comment:
I don't think a contributor has the right to add labels on GH, this is our job of maintainer